scripts/publish-charts.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+GIT_REPO_ROOT=$(git rev-parse --show-toplevel)
+BUILD_DIR="${GIT_REPO_ROOT}/chartbuild"
+TOOLS_DIR="${BUILD_DIR}/tools"
+CHARTS="${GIT_REPO_ROOT}/deploy/charts"
+PACKAGE_DIR="${GIT_REPO_ROOT}/chartbuild"
+export PATH="${TOOLS_DIR}:${PATH}"
+VERSION="$(git describe --tags --always)"
+
+if echo "${VERSION}" | grep -Eq "^v[0-9]+(\.[0-9]+){2}$"; then
+    git fetch --all
+    git config user.name "$GITHUB_ACTOR"
+    git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+    git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPO}
+    git config pull.rebase false
+    git checkout gh-pages
+    mv -n $PACKAGE_DIR/charts/*.tgz .
+    helmv3 repo index . --url https://${GITHUB_REPO_OWNER}.github.io/${GITHUB_REPO_NAME}
+    git add .
```

Comment:
Nit: `git add ./*.tgz` as a safety net?

Reply:
I also explicitly added the chart index.